DashAI/back/api/api_v1/endpoints/datasets.py

```diff
@@ -58,47 +64,89 @@ def parse_params(params):
 
 
 @router.get("/")
-async def get_dataset():
+async def get_datasets():
+    """
+    Returns all the available datasets in the database.
+
+    Returns
+    -------
+    List[JSON]
+        List of dataset JSONs
+    """
+
+    try:
+        all_datasets = session.query(models.Dataset).all()
+        if not all_datasets:
+            raise HTTPException(
+                status_code=status.HTTP_404_NOT_FOUND, detail="No datasets found"
+            )
+    except exc.SQLAlchemyError as e:
+        log.error(e)
```

Comment:
I believe that here it would be better to put an Exception rather than an error, since it is more serious than an error the fact that it is not possible to connect to the database.
```suggestion
        log.exception(e)
```

Additional question: why not use logger instead of log?